contrib/IECoreAlembic/src/IECoreAlembic/AlembicInput.cpp

```diff
@@ -359,44 +360,28 @@ Imath::M44d AlembicInput::transformAtTime( double time ) const
 	return result;
 }
 
-IECore::ToCoreConverterPtr AlembicInput::converter( IECore::TypeId resultType ) const
-{
-	return FromAlembicConverter::create( m_data->object, resultType );
-}
-
 IECore::ObjectPtr AlembicInput::objectAtSample( size_t sampleIndex, IECore::TypeId resultType ) const
 {
-	FromAlembicConverterPtr c = FromAlembicConverter::create( m_data->object, resultType );
-	if( c )
-	{
-		c->sampleIndexParameter()->setNumericValue( sampleIndex );
-		return c->convert();
-	}
-	return 0;
+	return ObjectAlgo::convert( m_data->object, sampleIndex, resultType );
 }
 
 IECore::ObjectPtr AlembicInput::objectAtTime( double time, IECore::TypeId resultType ) const
 {
-	FromAlembicConverterPtr c = FromAlembicConverter::create( m_data->object, resultType );
-	if( !c )
-	{
-		return 0;
-	}
-
 	size_t index0, index1;
-	double lerpFactor = sampleIntervalAtTime( time, index0, index1 );
+	const double lerpFactor = sampleIntervalAtTime( time, index0, index1 );
```

Comment:
It doesn't _need_ to be const, but in my old age I need all the help I can get from the compiler, so I'm tending towards everything being const unless it needs to be otherwise. I also find I'm quicker to understand a function if stuff is const - it's just one less thing to mentally track state for. Trivial in this case, sure, but I'm finding it helpful in the general case.